### PR TITLE
task-9360: asset screen: create ui

### PR DESCRIPTION
**Pull request by vanpd-0859**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/9360
### Goals :soccer:

https://marvelapp.com/3feje8d/screen/54841968
- Logo image + amount, value, price, 24h change label
- Send button
- Receive button
- Asset address label
- Copy image button
- Transaction history tableview:
+ Transaction cell: type image + transaction type, recepient, amount label
- Graph nav image button

### Implementation Details :construction:

Screen Shot 2019-04-12 at 4 31 28 PM

### CODE REVIEW

**Comment by vanpd-0859** [comment]
ready

File: CryptoWallet/Scenes/Home/Wallet/Asset/AssetViewController.swift [hunk 1]
```diff
@@ -10,14 +10,168 @@ import UIKit
...
 }
 
+extension AssetViewController: UITableViewDataSource, UITableViewDelegate {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return transactionDateList.count
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return transactionList.filter {
+            let transactionTime = Date(timeIntervalSince1970: $0.timeStamp)
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Home/Wallet/Asset/AssetViewController.swift]: Bỏ vào extension Date

File: CryptoWallet/Scenes/Home/Wallet/Asset/TransactionCell.swift [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            case .sent, .pending, .transferTokenTo(token: _):
+                $0.text = "-" + String(format: "%.3f", amount) + " " + symbol
+            default:
+                $0.text = String(format: "%.3f", amount) + " " + symbol
+            }
+        }
+    }
+}
+
+extension TransactionCell: Reusable {
```
Comment (tienpx3010) [inline CryptoWallet/Scenes/Home/Wallet/Asset/TransactionCell.swift]: Không cần đưa ra riêng ra một extension khi mình k mở rộng gì cả.

**Comment by vanpd-0859** [comment]
ready

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**